cql3/operation.hh

```diff
@@ -86,6 +86,10 @@ public:
      */
     virtual void execute(mutation& m, const clustering_key_prefix& prefix, const update_parameters& params) = 0;
 
+    const std::optional<expr::expression>& get_expression() const {
+        return _e;
+    }
```

Comment:
This works, but only by accident, because all of our operations happen to have on expression.

The serialization should be done in the sub-classes and the entry point should be a virtual function.

Reply:
Now `operation` modifies `update_query` and `get_expression` is unnecessary. Operations are prepared as before, in `modification_statement` and in the broadcast tables query's case it updates `update_query`.